.github/workflows/test-bdk-ffi-latest.yaml

```diff
@@ -0,0 +1,283 @@
+      - name: "Checkout"
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+          persist-credentials: false
+          fetch-depth: 0
+
+      - name: "Update bdk-ffi submodule to latest"
+        run: |
+          cd ./bdk-ffi/ \
+          && git fetch origin \
+          && git checkout master \
+          && git pull origin master \
+          && echo "Testing commit: $(git log -1 --pretty=format:'%h %s (author: %cn)')"
+
+      - name: "Cache"
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            ./target
+          key: ${{ runner.os }}-${{ hashFiles('**/Cargo.toml','**/Cargo.lock') }}
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python }}
+
+      - name: "Generate bdk.py and binaries"
+        run: bash ./scripts/generate-macos-x86_64.sh
+
+      - name: "Install build"
+        run: pip3 install build
+
+      - name: "Build wheel"
+        run: python3 -m build --wheel --config-setting=--build-option=--plat-name=macosx_11_0_x86_64 --verbose
+
+      - name: "Install wheel"
+        run: pip3 install ./dist/*.whl
+
+      - name: "Run tests"
+        run: python3 -m unittest discover --start "./tests/" --pattern "test_offline_*.py" --verbose
+
+      - name: "Upload artifact test"
+        uses: actions/upload-artifact@v4
+        with:
+          name: bdkpython-macos-x86_64-${{ matrix.python }}
+          path: /Users/runner/work/bdk-python/bdk-python/dist/*.whl
+
+  test-bdk-ffi-latest-windows-wheels:
+    name: "Build and run unittest on Windows wheels against the latest commit on bdk-ffi"
+    runs-on: windows-2022
+    strategy:
+      matrix:
+        python:
+          - "3.10"
+          - "3.11"
+          - "3.12"
+          - "3.13"
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+          persist-credentials: false
+          fetch-depth: 0
+
+      - name: "Update bdk-ffi submodule to latest"
+        run: |
+          cd ./bdk-ffi/; 
+          git fetch origin; 
+          git checkout master; 
+          git pull origin master; 
+          echo "Testing commit: $(git log -1 --pretty=format:'%h %s (author: %cn)')"
+
+      - name: "Cache"
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            ./target
+          key: ${{ runner.os }}-${{ hashFiles('**/Cargo.toml','**/Cargo.lock') }}
+      
+      - name: "Install Python"
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python }}
+
+      - name: "Generate bdk.py and binaries"
+        run: bash ./scripts/generate-windows.sh
+
+      - name: "Install build"
+        run: pip install build
+
+      - name: "Build wheel"
+        run: python -m build --wheel --verbose
+
+      - name: "Upload artifact test"
```

Comment:
I'm not sure we need to upload these artifacts for the jobs. Did you have a use for them in mind?